test/failure_handler/src/share/classes/jdk/test/failurehandler/action/PatternAction.java

```diff
@@ -59,6 +59,9 @@ public ProcessBuilder prepareProcess(HtmlSection section,
         for (int i = 0, n = args.length; i < n; ++i) {
             args[i] = args[i].replace(pattern, value) ;
         }
+        for (int i = 0, n = args.length; i < n; ++i) {
+            args[i] = args[i].replace("%java", helper.findApp("java").getAbsolutePath());
```

Comment:
are we sure that `java` from `PATH` (which is what `findApp` returns) is the right `java`?